AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientTestBase.swift

```diff
@@ -11,28 +11,32 @@ import AWSCognitoIdentityProvider
 import AWSTestResources
 
 class AWSMobileClientTestBase: XCTestCase {
-
+    
     static var cognitoIdentity: AWSCognitoIdentity!
     static var userPoolsAdminClient: AWSCognitoIdentityProvider!
     
     static var userPoolId: String!
     static var sharedEmail: String!
     static var identityPoolId: String!
-
+    
     // Test password that conforms to default UserPool password policies
     let sharedPassword: String = "Abc123@@!!"
```

Comment:
Should we move to the configuration file?

Reply:
Yeah, probably so. I'll move in a subsequent PR